### ISSUE TITLE
Bug with parser if '}' inside key/value entry

### ISSUE THREAD

**Issue by achaussier** [peakwinter/python-nginx#7]
Hello,

First, thanks for this project.
I submit the issue here, due to a connection issue with your Gitlab instance.

**Context**
The '}' char is inside a key/value state, inside a location of an existing config file

**Steps to reproduce**
1. Create a configuration file
```
$ cat /tmp/foo.conf
server {
    listen 80;
    location / {
        fastcgi_param MY_KEY "W*}4";
    }
}
```
2. Try to parse this file
```
 python-nginx  ~  22:38:31  python
Python 2.7.10 (default, Oct 14 2015, 16:09:02) 
[GCC 5.2.1 20151010] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import nginx
>>> nginx.loadf('/tmp/foo.conf')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/alexandre/.local/share/virtualenvs/python-nginx/local/lib/python2.7/site-packages/nginx.py", line 487, in loadf
    return load(f)
  File "/home/alexandre/.local/share/virtualenvs/python-nginx/local/lib/python2.7/site-packages/nginx.py", line 477, in load
    return loads(fobj.read())
  File "/home/alexandre/.local/share/virtualenvs/python-nginx/local/lib/python2.7/site-packages/nginx.py", line 449, in loads
    if isinstance(lopen[0], Server):
IndexError: list index out of range
>>> 
```
**Expected result**
Parser see the '}' as a part on key value statement and not the end of a block

**Comment by peakwinter** [comment]
Thanks! This will be fixed in v1.1.

**Comment by achaussier** [comment]
Thanks,

Have a nice day
